packages/node-firebird-driver/README.md

```diff
@@ -18,6 +18,31 @@ There is also plan to create a socket-based (pure Node.js) driver.
 
 yarn add node-firebird-driver
 
+## Contributing
+- Clone this repo
+- `lerna bootstrap` (lerna must installed globally)
+- `lerna link` (to link dependecies)
+- `lerna run build` (to build all packages)
+
+## Testing
+- `yarn run test`
+(default user and password will be used during tests)
+
+You can control test environment by adding a configuration file to the root of this project named (test-cfg.json).
+
+```
+// node-firebird-drivers/test-cfg.json
+{
+	"user": "sysdba",
+	"pw": "masterkey",
+	"role": "",
```

Comment:
And I'm even more unsure about `role`. It seems as test-specific configuration, not global.